lib-bee/package.json

```diff
@@ -5,7 +5,13 @@
     "main": "dist/index.js",
     "scripts": {
       "build":"tsc -p .",
-      "test": "echo \"Error: no test specified\" && exit 1"
+      "build:test":"echo skipped",
+      "test:browser": "echo skipped",
+      "test:node": "echo skipped",
+      "test": "echo skipped",
+      "integration-test:browser": "echo skipped",
+      "integration-test:node": "echo skipped",

```

Comment:
Thanks for the PR!
Seems like the tests are skipped for time being.
Can you refactor the tests file to have `.spec.ts` extension and have it under a `test` folder?
You can merge this in and create a new PR for above.
Can include the build outputs as well.

Reply:
cool

Reply:
so the tests in lib-bee are skipped,but we're running in lib-aaa. We're not concerned about lib-bee tests at all